File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 29/11/2025, 19:22:41
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i=0;i<nums.size();i++){
            for(int j=i+1;j<nums.size();j++){
                if((nums[i]+nums[j])==target){
                   return {i,j}; 
                }
            }

        
        }
      return {};  
    }
    
};

File: 1-TwoSum/1-TwoSum.py
# Last updated: 02/01/2026, 02:14:10
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
            for i in range(len(nums)):
                for j in range(i+1,len(nums)):
                    if nums[i]+nums[j]==target :
                         List=[]
                         List.append(i)
                         List.append(j)
                         return List
                    